build/atuin/build.sh

```diff
@@ -17,7 +17,7 @@
 . ../../lib/build.sh
 
 PROG=atuin
-VER=16.0.0
+VER=17.0.0
```

Comment:
In the meantime version `17.0.1` has been released. Would you mind bumping it to the latest release?